INSTALL

```diff
@@ -211,64 +193,74 @@ IP addresses:
 
 You can now install the Eucalyptus RPMs onto each system.
+The Eucalyptus documentation can be found here:
+http://docs.hpcloud.com/eucalyptus
 
-Follow the instructions from the Installation Guide for all of the steps to
-plan and configure your system's dependencies, before the steps describing
-how to install Eucalyptus or nightly release packages. The Eucalyptus
-documentation can be found here:
+Follow the instructions in the Installation Guide for the following
+sections:
 
-http://docs.hpcloud.com/eucalyptus
+- Plan Your Installation
+- Configure Dependencies
+
+After completing those sections, install the packages you have built.
+Install ALL of the Eucalyptus RPMs even if all the components are not
+needed, to prevent dependency problems with Eucalyptus packages that have
+the same names but different version numbers in the eucalyptus-nightly
+repository. You can still choose which components to configure and run on
+each system.
 
-At this point, instead of installing the packages as described in the
-Installation Guide, install the packages you have built. Install all of the
-Eucalyptus RPMs even if all the components are not needed, to prevent
-dependency problems with Eucalyptus packages that have the same names but
-different version numbers in the eucalyptus-nightly repository. You can
-still choose which components to configure and run on each system.
+If you are updating an existing deployment, stop all eucalyptus services
+before re-installing packages.
 
 For the build system (if you're installing onto it):
 
 >>>>> Command 6a: 
 # yum install $EUCARPM/RPMS/noarch/* $EUCARPM/RPMS/x86_64/*
+Note: If you have rebuilt packages per Step 7 below, the package versions
+will likely be the same, so "yum install" will do nothing. To force the
+new packages to be updated, you must re-install them:
+# yum reinstall $EUCARPM/RPMS/noarch/* $EUCARPM/RPMS/x86_64/*
 
 For all other systems:
 
 >>>>> Command 6a: 
 # yum install $EUCAPKGS/noarch/* $EUCAPKGS/x86_64/*
+Or, if you are updating packages per above:
+# yum reinstall $EUCAPKGS/noarch/* $EUCAPKGS/x86_64/*
 
 The yum command may install many other Linux packages on which the
 Eucalyptus packages depend.
 
+Continue following the instructions from the Installation Guide, starting at
+the "Install Eucalyptus Release Packages" section (NOT the "Install Nightly
+Release Packages" section).
 
-7. Configure Eucalyptus  (on all systems)
-=======================
-
-From this point forward, follow the instructions from the Installation
-Guide, starting at the "Install Eucalyptus Release Packages" section. The
-Eucalyptus documentation can be found here:
-
-http://docs.hpcloud.com/eucalyptus
-
-Skip the "yum install" commands for the packages you have already installed,
-including:
-
-eucalyptus-cc 
+Skip the "yum install" commands for what you've already installed:
+- the package repositories from downloads.eucalyptus.com
+- the packages you have already installed:
```

Comment:
These dashes confuse me, are they 'list items'?  Is that saying skip the yum install commands on these two items, or these two items PLUS the packages below (eucalyptus-node et. al.)